app/src/main/java/com/sunasterisk/youtubebackground/utils/VideoPlayService.kt

```diff
@@ -0,0 +1,115 @@
+package com.sunasterisk.youtubebackground.utils
+
+import android.app.Service
+import android.content.Intent
+import android.os.Binder
+import android.os.IBinder
+import android.widget.Toast
+import com.google.android.youtube.player.YouTubeInitializationResult
+import com.google.android.youtube.player.YouTubePlayer
+import com.sunasterisk.youtubebackground.data.model.Video
+import com.sunasterisk.youtubebackground.screen.video.VideoActivity
+import kotlinx.android.synthetic.main.activity_video.*
+
+class VideoPlayService : Service(), YouTubePlayer.PlayerStateChangeListener,
+    YouTubePlayer.PlaylistEventListener {
+
+    val binder: IBinder = PlayBinder()
+    var view: VideoActivity? = null
+    var youTubePlayer: YouTubePlayer? = null
+    var possiton: Int = 0
```

Comment:
check lại access modifier cho các biến
```suggestion
    var position = 0
```